ICAFIX/scripts/icaDim.m

```diff
@@ -58,7 +59,8 @@
     if VN~=0
       noise_unst = (u(:,Out.VNDIM(c):Out.DOF)*EigS(Out.VNDIM(c):Out.DOF,Out.VNDIM(c):Out.DOF)*v(:,Out.VNDIM(c):Out.DOF)')';
       Out.noise_unst_std = max(std(noise_unst,[],2),0.001); clear noise_unst;
-      data_detrend_vn = (u*EigS*v')' ./ repmat(Out.noise_unst_std,1,size(data_trend,2));
+      data_detrend_vn = (u*EigS*v')';
+      data_detrend_vn = data_detrend_vn ./ repmat(Out.noise_unst_std,1,size(data_trend,2));
```

Comment:
Breaking expressions can't help when there are less than 3 matrices on the right hand side, even if matlab were completely stupid about evaluating them.

Reply:
Actually, this line should be beneficial, somehow I wasn't thinking of `(u*EigS*v')'` as a computation.